mitmproxy/contrib/kaitaistruct/dtls_client_hello.py

```diff
@@ -0,0 +1,203 @@
+# This is a generated file! Please edit source .ksy file and use kaitai-struct-compiler to rebuild
```

Comment:
Can you please add the corresponding .ksy file, or alternatively get this into https://github.com/kaitai-io/kaitai_struct_formats/tree/master/network (I'm sure they woud appreciate it) and https://github.com/mitmproxy/mitmproxy/blob/main/mitmproxy/contrib/kaitaistruct/make.sh?

We also need to compile the ksy with the latest Kaitai compiler as mitmproxy does not always ship with `pkg_resources` (https://github.com/mitmproxy/mitmproxy/issues/5401).